server/utils/referenceDataCodeUtils.ts

```diff
@@ -0,0 +1,11 @@
+export const formatMedicalDietaryRequirements = (requirements: { id: string; description: string }[]): string[] => {
```

Comment:
I've left this in for now, though the `FREE_FROM` label is likely to disappear in the future it's there at the moment - but this can be removed/made more generic once we've got the dietary requirements confirmed